packages/genesys-spark-components/src/components/beta/gux-time-zone/gux-time-zone.service.ts

```diff
@@ -0,0 +1,19 @@
+import { GuxTimeZoneIdentifier } from '../../../i18n/time-zone/types';
+import { getTimeZoneList, formatOffset } from '../../../utils/date/time-zone';
+import { GuxTimeZoneListing } from '../gux-time-zone-picker/gux-time-zone-picker.types';
+
+export function shortenZone(zone: string): string {
```

Comment:
Is there any way to document here an example of a zone string input and how it'd be transformed to then be returned?

Reply:
This component has already been in the code base before, so there isn't new code to review in this PR. Any changes to the component itself (if needed) should be in another PR. This is to avoid potentially introducing bugs at this stage.